package.json

```diff
@@ -21,13 +21,16 @@
     "@angular/platform-browser": "~2.1.0",
     "@angular/platform-browser-dynamic": "~2.1.0",
     "@angular/router": "~3.1.0",
+    "angular-calendar": "~0.4.0",
     "core-js": "^2.4.1",
+    "font-awesome": "^4.7.0",
```

Comment:
Font awesome was already in the project. But your way of including it is better. Please remove it from index.html.